apps/prairielearn/src/pages/instructorAssessmentStatistics/instructorAssessmentStatistics.ts

```diff
@@ -1,26 +1,35 @@
-// @ts-check
 import * as express from 'express';
-const asyncHandler = require('express-async-handler');
+import asyncHandler = require('express-async-handler');
```

Comment:
I believe this is changing with the move to ESM, will likely be captured by the merge conflict, but just pointing out.

Reply:
Updated in merge [f903993](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/f903993f70436b07f9d110f5ad33ef7a39acccf7).